.github/workflows/rust-openssl.yml

```diff
@@ -0,0 +1,35 @@
+# GitHub Actions workflow to run rust-openssl regress tests.
+name: "rust-openssl"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *" # At 00:00 daily.
+
+jobs:
+  test:
+    name: "Test"
+    runs-on: "ubuntu-latest"
+    permissions:
+      contents: read
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v4
+
+      - name: "Build LibreSSL"
+        run: |
+          ./autogen.sh
+          ./configure --prefix="${HOME}/opt"
+          make all install
+
+      - name: "Clone rust-openssl"
+        run: |
+          git clone https://github.com/sfackler/rust-openssl.git
+
+      - name: "Run rust-openssl tests"
+        run: |
+          cd rust-openssl
+          OPENSSL_DIR=${HOME}/opt
+          LD_LIBRARY_PATH=${HOME}/opt/lib
+          patch -p1 < ../.github/rust-openssl.patch
+          cargo test
```

Comment:
Could you change this into `cargo test --verbose`, so we can see what's going on please?

Reply:
Sure